### PR TITLE
Merged /chain/height and /chain/score into /chain/info and included finalization information

### PR DESCRIPTION
**Pull request by Vektrat**
as discussed here: https://github.com/nemtech/catapult-rest/issues/461

### CODE REVIEW

**Comment by Vektrat** [comment]
@AnthonyLaw

**Comment by AnthonyLaw** [comment]
@Vektrat It look good for me 👍

**Comment by Vektrat** [comment]
please review again, i left the specific types (FinalizationPoint/Epoch) because they are reused and serve as a better documentation, check if the type `integer` is fine, or you prefer to specify it to uint32 somehow

File: spec/schemas/_index.yml [hunk 1]
```diff
@@ -13,6 +13,8 @@ CosignatureVersion:
   "$ref": "./CosignatureVersion.yml"
 Difficulty:
   "$ref": "./Difficulty.yml"
+FinalizationEpoch:
+  "$ref": "./FinalizationEpoch.yml"
 FinalizationPoint:
```
Comment (rg911) [inline spec/schemas/_index.yml:18]: old property still here

**Review by rg911 — CHANGES_REQUESTED** [review]

**Review by rg911 — DISMISSED** [review]

File: spec/core/chain/schemas/ChainInfoDTO.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+type: object
+required:
+  - height
+  - finalizedHeight
```
Comment (fboucquez) [inline spec/core/chain/schemas/ChainInfoDTO.yml:4]: is `finalizedHeight` duplicated value? User can get it from `latestFinalizedBlock.height`

File: spec/core/chain/schemas/FinalizedBlockDTO.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+type: object
```
Comment (fboucquez) [inline spec/core/chain/schemas/FinalizedBlockDTO.yml:1]: Is this the same schema than the finalized block WS payload? We should move this file to the top level schema folder

File: spec/core/chain/schemas/ChainInfoDTO.yml (hunk 2 above)
Comment (fboucquez) [reply spec/core/chain/schemas/ChainInfoDTO.yml]: @Vektrat

**Review by fboucquez — APPROVED** [review]

**Pull request merged**